spec/pb_kits/playbook/kits/radio_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require_relative "../../../../app/pb_kits/playbook/pb_radio/radio"
+
+RSpec.describe Playbook::PbRadio::Radio do
+  subject { Playbook::PbRadio::Radio }
+
+  it { is_expected.to define_partial }
+
+  # Do not leave this file blank. Use other spec files for example tests.
+
+end
```

Comment:
You need to fill this out. Ask @thestephenmarshall for help guidance, but simple class tests for props is a good start